README.md

```diff
@@ -67,17 +67,17 @@ Review our [Contributing Instructions](CONTRIBUTING.md) before beginning editing
 | 48 | 2022-xx-xx | Clair Sullivan| [Arrays, Linked Lists and Graphs](2022/48-clair-linkedlist.md)       |                        |              |        |
 | 49 | 2022-xx-xx | Lauren Burke| [Setting up a Personal Website with Jekyll & GitHub Pages](2022/49-lauren-jekyll-blog.md)       |                        |              |        |
 | 50 | 2022-xx-xx | Julia Signell| [Introduction to Holoviz](2022/50-julia-holoviz.md)       |                        |              |        |
-| 51 | 2022-xx-xx | Rob Lanphier| [Editing Wikipedia: Because Someone Has to...](2022/51-rob-wikipedia.md)|                        |              |        |
-| 52 | 2022-xx-xx | William Lyon| [Introduction to GraphQL for Data Scientists](2022/52-will-graphql.md)|
-| 53 | 2022-xx-xx | Sebastian Raschka and Adrian Wälchli| [Introduction to PyTorch](2022/53-sebastian-adrian-pytorch.md) | 
-| 54 | 2022-xx-xx | Melissa Weber Mendonça | [Contributing to SciPy](2022/54-melissa-scipy.md) | 
-| 55 | 2022-xx-xx | Joe Torreggiani | [Intro to Django](2022/55-joe-django.md) | 
-| 56 | 2022-xx-xx | Tereza Iofciu | [Short Stories of Data Visualization](2022/56-tereza-visualization.md) | 
+| 51 | 2022-04-05 | Rob Lanphier| [Editing Wikipedia: Because Someone Has to...](2022/51-rob-wikipedia.md)| [v51_Editing_Wikipedia](https://www.youtube.com/watch?v=s5kUSMUaZLM)                     |              |        |
+| 52 | 2022-04-26 | William Lyon| [Introduction to GraphQL for Data Scientists](2022/52-will-graphql.md)| [v52_GraphQL_for_Data_Scientists](https://youtu.be/3QVguMgyAhU)
+| 53 | 2022-05-10 | Sebastian Raschka and Adrian Wälchli| [Introduction to PyTorch](2022/53-sebastian-adrian-pytorch.md) | [v53_PyTorch_introduction](https://www.youtube.com/watch?v=B5GHmm3KN2A)
```

Comment:
This event also has its own playlist.  We should replace the video link with the playlist.